.grenrc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    "dataSource": "prs",
+    "prefix": "",
+    "onlyMilestones": false,
+    "ignoreCommitsWith": ["chore", "refactor", "style"],
+    "ignoreIssuesWith": ["no-release"],
+    "ignoreTagsWith": ["-rc", "-alpha", "-beta", "test", "current"],
+    "ignoreLabels": ["closed", "automation", "enhancement", "bug", "fix",
```

Comment:
Looks like most of the labels mentioned in groupBy are ignored?
What does `ignoreLabels` mean? and `ignoreCommitsWith` also? As commit with `chore` is included in group by as `Minor` fixes.

Reply:
Yes. We don't need `ignoreCommits` explicitly but we need `ignoreLabels` to format output of notes. else it will look like this: 

<img width="812" alt="Screenshot 2021-01-21 at 4 05 39 PM" src="https://user-images.githubusercontent.com/26570044/105340259-a9e5d080-5c03-11eb-9451-15f407d840df.png">